geode-wan/src/distributedTest/java/org/apache/geode/internal/cache/wan/wancommand/ListGatewaysCommandDUnitTest.java

```diff
@@ -450,6 +450,30 @@ public void testListGatewaysReceiversOnlyFalseAndSendersOnlyFalseReturnsSendersA
         .hasRowSize(expectedGwReceiverSectionSize).hasColumns().contains("Port", "Member");
   }
 
+  @Test
+  public void testListGatewaysWithOneDispatcherThread() {
+    String command =
+        "create gateway-sender --id=ln_Serial --remote-distributed-system-id=2 --dispatcher-threads=1";
```

Comment:
Instead of hard-coding the gfsh command, would it be possible to use `CommandStringBuilder` with the static strings found in `CliStrings` here?